kernel/packages/shared/chat/sagas.ts

```diff
@@ -105,12 +105,16 @@ function* handleReceivedMessage(action: MessageReceived) {
 function* handleSendMessage(action: SendMessage) {
   const { body: message } = action.payload
 
-  let entry: ChatMessage | null = null
+  let entry: ChatMessage | null | undefined = null
 
   // Check if message is a command
   if (message[0] === '/') {
     entry = handleChatCommand(message)
 
+    if (entry === undefined) { // Command is found but has no feedback message
```

Comment:
I rather filter empty body messages instead of adding `undefined | null` response types.

https://hinchman-amanda.medium.com/null-pointer-references-the-billion-dollar-mistake-1e616534d485

```suggestion
    if (entry && entry.body.length == 0) { // Command is found but has no feedback message
```

Reply:
Much better! nice suggestion